### PR TITLE
Add hackathon 2019 1 report

### PR DESCRIPTION
**Pull request by Fohlen**

### CODE REVIEW

File: content/blog/2019/03-04-first-hackathon-2019-report.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+![Inexor people hacking](https://raw.githubusercontent.com/inexorgame/blog-media/master/2019/03/inexor_hacking_from_above.jpg "inexor people hacking")
+
+On Saturday we held some quick talks about current state-of-the art Inexor which you can find in our [media repository](https://github.com/inexorgame/blog-media/tree/master/2019/03).
+
+Let's dive right into what we accomplished this Hackathon:
+
+### User stories
+
+`@martinmuzatko` and `@koppiman` wrote a myrad of usefull user stories, sorted and prioritized them, so that it will be straightforward to implenent a _"minimum viable prototype"_ from this list.
```
Comment (MartinMuzatko) [inline content/blog/2019/03-04-first-hackathon-2019-report.md]: It is myriad

File: content/blog/2019/03-04-first-hackathon-2019-report.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+![CCCFR](https://raw.githubusercontent.com/inexorgame/blog-media/master/2019/03/Hackspace_CCCFR.jpeg "CCCFR")
+
+Firstly we would like to thank the CCCFR for hosting us in their lovingly hackerspace. It's been yet one of the most homelike locations that we've been to.
+
+Also thanks to tau for preparing his famous _Taulasch_ for us, it was very delicious :relaxed:
+
+## What we did
+
+We had lot's of fun cooking, prototyping, exploring Freiburg, and as well `@Fohlen` cut some cucumbers really well.
```
Comment (MartinMuzatko) [inline content/blog/2019/03-04-first-hackathon-2019-report.md]: Unnecessary apostrophe

Comment (MartinMuzatko) [reply content/blog/2019/03-04-first-hackathon-2019-report.md]: Maybe more context for the cucumbers? XD

File: content/blog/2019/03-04-first-hackathon-2019-report.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### User stories
+
+`@martinmuzatko` and `@koppiman` wrote a myrad of usefull user stories, sorted and prioritized them, so that it will be straightforward to implenent a _"minimum viable prototype"_ from this list. 
+
+### User interfaces
+
+`@martinmuzatko` wrote a nice user interface for the [entity-system]() which you can see below:
+
+![Entity UI](https://raw.githubusercontent.com/inexorgame/blog-media/master/2019/03/entity_ui.png "Entity UI")
```
Comment (MartinMuzatko) [inline content/blog/2019/03-04-first-hackathon-2019-report.md]: Lets without apostrophe

> Lets without an apostrophe is the third-person singular form of the verb let, meaning “to allow or permit”: She lets the dog out every morning. Let's with an apostrophe is a contraction of “let us,” which is used in all varieties of speech and writing to introduce a suggestion or request: Let us consider all the requests

File: content/blog/2019/03-04-first-hackathon-2019-report.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Let's dive right into what we accomplished this Hackathon:
+
+### User stories
+
+`@martinmuzatko` and `@koppiman` wrote a myriad of usefull user stories, sorted and prioritized them, so that it will be straightforward to implenent a _"minimum viable prototype"_ from this list. 
+
+### User interfaces
+
+`@martinmuzatko` wrote a nice user interface for the [entity-system]() which you can see below:
```
Comment (a-teammate) [inline content/blog/2019/03-04-first-hackathon-2019-report.md]: maybe we should mention that it is a developer user interface?

File: content/blog/2019/03-04-first-hackathon-2019-report.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+We also discussed what can be done in the future and came up with an agenda until our next Hackathon:
+
+- Implement the basic connectors and processors as defined in the [specification](https://inexor.org/wiki/features/Entity-System.html) including the basic `REST-API`
+- Implement a first prototype of an octree-based world rendering
+- Think about thread pooling and ideally implement a solution
+- Think about synchronisation (multiplayer)
+- Finish the work on the basic UI for the `entity-system`
+- `@koppiman` will talk with Medienschule Salzburg about a potential model contest for the first Inexor demo
+
+We would like to thank everyone who was involved, helped, discussed and had fun with us!
```
Comment (a-teammate) [inline content/blog/2019/03-04-first-hackathon-2019-report.md]: mention regina here?

File: content/blog/2019/03-04-first-hackathon-2019-report.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+It may seem like a small step, but this demoscene lays foundation for a huge change in how games will work in the future, or as `@aschaeffer` put it: _"What can't be done with the entity-system? I've thought about this for quite some time .... and to be honest, there's nothing I can come up with, that we can't make an entity"_
+
+### Future to-do
+
+We also discussed what can be done in the future and came up with an agenda until our next Hackathon:
+
+- Implement the basic connectors and processors as defined in the [specification](https://inexor.org/wiki/features/Entity-System.html) including the basic `REST-API`
+- Implement a first prototype of an octree-based world rendering
+- Think about thread pooling and ideally implement a solution
```
Comment (a-teammate) [inline content/blog/2019/03-04-first-hackathon-2019-report.md]: "Implement a safer concurrency solution for everything outside the Entity System's graph"

**Pull request merged**